fundingmanager.go

```diff
@@ -943,6 +943,32 @@ func (f *fundingManager) stateStep(channel *channeldb.OpenChannel,
 		fndgLog.Debugf("Channel(%v) with ShortChanID %v: successfully "+
 			"added to router graph", chanID, shortChanID)
 
+		// Give the caller a final update notifying them that
+		// the channel is now open.
+		// TODO(roasbeef): only notify after recv of funding locked?
+		fundingPoint := channel.FundingOutpoint
+		cp := &lnrpc.ChannelPoint{
+			FundingTxid: &lnrpc.ChannelPoint_FundingTxidBytes{
+				FundingTxidBytes: fundingPoint.Hash[:],
+			},
+			OutputIndex: fundingPoint.Index,
+		}
+		upd := &lnrpc.OpenStatusUpdate{
```

Comment:
Nit: can limit the scope of this to be within the if statement below.

Reply:
Yep 👍 https://github.com/lightningnetwork/lnd/pull/1910/commits/479f80dc7040a92c2a27baa177fdac89e8ca675a